core/sql/executor/HBaseClient_JNI.h

```diff
@@ -576,6 +550,30 @@ class HBaseClient_JNI : public JavaObjectInterface
   HBC_RetCode incrCounter( const char * tabName, const char * rowId, const char * famName, 
```

Comment:
Could incrCounter method benefit from the new change? Or maybe it is already getting is own htc and deleting it at the end?